cage.c

```diff
@@ -61,6 +61,8 @@
 #include "xwayland.h"
 #endif
 
+extern bool numlock;
```

Comment:
A field in `struct cg_server` is generally used here for such a purpose instead of global variable.